content/blog/survey-02/index.md

```diff
@@ -0,0 +1,221 @@
++++
+title = "Newsletter Survey Results"
+date = 2024-06-03
+transparent = true
+draft = false
++++
+
+Since we are [rebooting the newsletter](https://gamedev.rs/blog/newsletter-changes/), we wanted to know more about our readers. 
+52 of you filled out the survey last month. Thank you very much!
+
+The biggest takeaways are:
+- People are generally excited about the newsletter
+- The current frequency of the newsletter is good
+- Readers do not want anything in the newsletter generated by AI
+- Contributing to the newsletter could be easier. If you've got ideas on how to make this happen, please [let us know](https://github.com/rust-gamedev/rust-gamedev.github.io/issues/1519)!
+
+We will now go through the results in the same order as the questions were asked. The full analysis and data is open-sourced on [GitHub](https://github.com/janhohenheim/rust-gamedev-statistics/tree/main/jan-hohenheim-2024).
+
+## Excitement
+
+![Excitement barplot](excitement.png)
+
+On average, readers are excited about the newsletter (the p-value for a t-test of mean 3 is 9.9e-5). The mean excitement level is 3.6 out of 5, the median is 4.
```

Comment:
A few statistical complaints:

1. A t-test is not particularly appropriate for ordinal data like this.
2. A null hypothesis of "3" feels weird: if "not excited" is a 1, shouldn't that be the null?

You could use a chi squared test here, but that's similarly useless practically, even if it's more statistically correct.

Instead, I would report standard deviation and sample size and drop the test here completely.

Reply:
1. Yeah, you're right.
2. I felt like neutral should be the null, as "people are not excited" seems more like an own hypothesis. However, this is my fault in the first place since in hindsight I should have used a Likert scale.

Done :)